src/ray/protobuf/pubsub.proto

```diff
@@ -43,3 +47,47 @@ message WorkerRefRemovedMessage {
   // still be borrowing an object ID that was nested inside.
   repeated ObjectReferenceCount borrowed_refs = 1;
 }
+
+///
+/// Subscribe
+///
+message Command {
+  ChannelType channel_type = 1;
```

Comment:
probably unnecessary as Yi pointed out before. I might remove it in the future PRs

Reply:
Thanks!